server/http_hooks.go

```diff
@@ -45,3 +51,66 @@ func (p *Plugin) handleStatus(w http.ResponseWriter, r *http.Request) {
 func (p *Plugin) handleHello(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("Hello World!"))
 }
+
+func (p *Plugin) handleEphemeralUpdate(w http.ResponseWriter, r *http.Request) {
+	request := model.PostActionIntegrationRequestFromJson(r.Body)
+	URL := fmt.Sprintf("%s", *p.API.GetConfig().ServiceSettings.SiteURL)
+
+	count := int(request.Context["count"].(float64)) + 1
+
+	post := &model.Post{
+		Id:        request.PostId,
```

Comment:
I don't think this is needed. I would rather not be explicit here, as most integrations arn't either.

Reply:
Id is needed so UpdateEphemeralPost knows which post to update.  (unless you meant count, which is used to show how many times the post was updated) .  Could you clarify what you meant please?

Reply:
Ah, I think you mean't count.  I updated it to this:
`count := request.Context["count"].(float64) + 1`